### PR TITLE
Port KeyboardAvoidingView, KeyboardAwareFlatList and ReadableContentView to @wordpress/components

### PR DESCRIPTION
**Pull request by Tug**
This PR is step 2 of our mobile refactor https://github.com/wordpress-mobile/gutenberg-mobile/issues/958

## Description
Its goal is to port some of our reusable components that live in `gutenberg-mobile` to `gutenberg` so we can reuse them in `gutenberg` and thus port our other components as well.

## How has this been tested?
Gutenberg mobile PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/1048

## Types of changes
This is a refactor PR

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/components/src/mobile/readable-content-view/style.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.container {
```
Comment (gziolo) [inline packages/components/src/mobile/readable-content-view/style.scss]: Filename should be updated to `style.native.scss` to ensure it never gets included for the web version.

Comment (Tug) [reply packages/components/src/mobile/readable-content-view/style.scss]: Thanks, updated!

**Review by pinarol — APPROVED** [review]
LGTM! Tested with the steps in parent PR

**Comment by Tug** [comment]
Thanks for the review @pinarol 🙇

**Pull request merged**